docs/pre-release/concepts/intercepts.md

```diff
@@ -0,0 +1,149 @@
+                    anim.finish();
+                    anim.play();
+                })
+                queueAnimation();
+            }, 3000);
+        };
+        queueAnimation();
+    }, el);
+    return (
+        <div ref={el} style="text-align: center">
+            <InterceptAnimationSVG style="max-width: 700px" {...props} />
+        </div>
+    );
+};
+
+export function TabsContainer({ children, ...props }) {
+    const [state, setState] = React.useState({curTab: "personal"})
+    return (
+        <div class="TabGroup">
+            <TabContext value={state.curTab}>
+                <AppBar class="TabBar" elevation={0} position="static">
+                    <TabList onChange={(ev, newTab) => {setState({curTab: newTab})}} aria-label="intercept types">
+                        <Tab class="TabHead" value="regular" label="No intercept"/>
+                        <Tab class="TabHead" value="global" label="Global intercept"/>
+                        <Tab class="TabHead" value="personal" label="Personal intercept"/>
+                    </TabList>
+                </AppBar>
+                {children}
+            </TabContext>
+        </div>
+    );
+};
+
+<TabsContainer>
+<TabPanel class="TabBody" value="regular">
+
+## No intercept
+
+<Animation class="mode-regular" />
+
+This is the normal operation of your cluster without Telepresence.
+
+</TabPanel>
+<TabPanel class="TabBody" value="global">
+
+## Global intercept
+
+<Animation class="mode-global" />
+
+**Global intercepts** replace the Kubernetes "Orders" service with the
+Orders service running on your laptop.  The users see no change, but
+with all the traffic coming to your laptop, you can observe and debug
+with all your dev tools.
+
+ 1. Intercept your service from your CLI:
+
+    ```shell
+    telepresence intercept SERVICENAME --http-match=all
+    ```
+
+    <Alert severity="info">
+
+    Make sure your current kubectl context points to the target
+    cluster.  If your service is running in a different namespace than
+    your current active context, use or change the `--namespace` flag.
+
+    </Alert>
+
+ 2. Send requests to your service:
+
+    All requests will be sent to the version of your service that is
+    running in the local development environment.
+
+</TabPanel>
+<TabPanel class="TabBody" value="personal">
+
+## Personal intercept
+
+<Animation class="mode-personal" />
+
+**Personal intercepts** allow you to be selective and intercept only
+some of the traffic to a service.
+
+**<span style="color: #f24e1e">Orange</span>** requests are being made
+by a developer on their laptop and the **<span style="color:
+#00c05b">green</span>** are made by a teammate on a different laptop.
+
+They can intercept the same service in the Kubernetes cluster to
+create a shared development environment.
+
+ 1. Intercept your service from your CLI:
+
+    ```shell
+    telepresence intercept SERVICENAME --http-match=Personal-Intercept=126a72c7-be8b-4329-af64-768e207a184b
```

Comment:
I would add a qualifier here saying "we're using `Personal-Intercept=126a72c7-be8b-4329-af64-768e207a184b` as the header, but you can use any <key>=<value> that you want" or something like that just so it's abundantly obvious that this value can, and should, change.